lib/importers/schema_dot_org_json_data.rb

```diff
@@ -67,4 +68,9 @@ def process_organization_name(job)
     return nil
   end
 
+  def has_minimum_required_props(job_posting)
+    required_properties = ['hiringOrganization','datePosted', 'url']
+    required_properties.map {|x| !job_posting[x].blank?}.all?
```

Comment:
You did, (see file above), this is another occurence. I missed this on my first pass through.

Might be good to refactor into a base class if they are the same function.